api-reference/10 UI Components/dxGantt/1 Configuration/sorting.md

```diff
@@ -1,10 +1,182 @@
 ---
 id: dxGantt.Options.sorting
 type: dxGanttSorting
+inheritsType: dxGanttSorting
 ---
 ---
 ##### shortDescription
-<!-- Description goes here -->
+Configures sorting settings.
```

Comment:
I'd remove '-ing' in this phrase. The option name contains 'sorting', but I'd try to use 'sort' in text where possible

```suggestion
Configures sort settings.
```